### ISSUE TITLE
Needless trailing whitespace in error messages

### ISSUE THREAD

**Issue by jidanni** [perltidy/perltidy#13]
No big deal but just want to let you know you are adding whitespace to the end of some lines.
```
$ cat -e perltidy.ERR
Perltidy version is 20181120$
$
12: my %a=(h=>{m=3785 > , x=3805},d=>{m=>76.1, x=>78.7});$
               -------------^                            $ << LINE WITH WHITESPACE
found Number where operator expected (previous token underlined)$
```

**Comment by shancock9** [comment]
Thanks, that will be fixed in the next release.

**Comment by shancock9** [comment]
This is fixed in version 20190915
